serverless/src/utils/slack-processor-util.ts

```diff
@@ -0,0 +1,84 @@
+import * as uuid from "uuid";
+import {WebClient} from '@slack/web-api';
+
+// Configure client
+const client = new WebClient(process.env.SLACK_OAUTH_TOKEN);
+const supportChannel: string = "support-channel";
```

Comment:
i think that this should be a environment variable that consumers can set.

Reply:
I agree I'll move this.